cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/securityContext.xml

```diff
@@ -21,24 +21,35 @@
 -->
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xmlns:p="http://www.springframework.org/schema/p"
-       xmlns:sec="http://www.springframework.org/schema/security"
+       xmlns:mvc="http://www.springframework.org/schema/mvc"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
-       http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd">
+       http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc.xsd">
+
+    <description>
+        Security configuration for sensitive areas of CAS : status and statistics.
+    </description>
+
+    <bean id="config" class="org.pac4j.core.config.Config">

```

Comment:
What is the current behavior now? 401 or 403? 

We have specific error pages that intercept those error code and show a "you are not authorized to see this page" message. Does this changeset work correctly to show that error?